app/DynamicsAdapter/Fams3Adapter.Dynamics/SearchRequest/SSG_SearchRequest.cs

```diff
@@ -10,5 +10,10 @@ public class SSG_SearchRequest
 
         [JsonProperty("ssg_name")]
         public string FileId { get; set; }
+
+        public override string ToString()
+        {
+            return SearchRequestId.ToString();
+        }
```

Comment:
I can't see where we use this for search request entity. do we use it ?

Reply:
yes, we need it when do concating the fields.